eZ/Publish/Core/MVC/Symfony/Security/EventListener/SecurityListener.php

```diff
@@ -213,6 +214,15 @@ public function onKernelRequest( GetResponseEvent $event )
             return;
         }
 
+        try
+        {
+            $this->repository->getUserService()->loadUser( $this->repository->getCurrentUser()->contentInfo->id );
+        }
+        catch ( NotFoundException $e )
+        {
+            $this->repository->setCurrentUser( $this->repository->getUserService()->loadUser( 10 ) );
```

Comment:
@crevillo Yes, it returns the new stack user object as usual, no problem there. This is the source of the bug actually: New stack doesn't know that the user has been deleted, and will happily continue to use the new stack user objects (until it discovers that the user has no role permissions).